modules/ROOT/pages/message-logging-policy.adoc

```diff
@@ -63,7 +65,27 @@ Error handler after API flow:
 
 image::apim-message-logging-policy-states-3.png[State after the error handler executes]
 
-== Logging Policy in XML
+
+== Configuring Policy Parameters
+
+When you apply the Message Logging policy to your API from the UI, you can configure the following parameters:
+
+image::apim-message-logging-policy-ui.png[Message Logging Policy UI]
+
+[%header%autowidth.spread]
+|===
+|Parameter |Description | Required
+|Message |A DataWeave expression to use for extracting the message to be logged.
+Note: If the payload is used as part of the message log, do not configure the Listener as non-repeatable (by default this setting is off).|Required
```

Comment:
```suggestion
Note: If the payload is used as part of the message log, do not configure the Listener as nonrepeatable (by default this setting is off).|Required
```
Do we need to say that this setting is off, esp. if telling them to NOT set it?

Reply:
Done! Thanks!